tests/FSharp.Compiler.ComponentTests/TypeChecks/Graph/CompilationFromCmdlineArgsTests.fs

```diff
@@ -43,16 +43,16 @@ module CompilationFromCmdlineArgsTests =
             for diag in diagnostics do
                 printfn "%A" diag
 
-            Assert.That(exitCode, Is.Zero)
+            Assert.Equal(exitCode, 0)
         finally
             Environment.CurrentDirectory <- oldWorkDir
 
-    [<TestCaseSource(nameof localProjects)>]
-    [<Explicit("Slow, only useful as a sanity check that the test codebase is sound and type-checks using the old method")>]
+    [<MemberData(nameof localProjects)>]
+    [<Theory(Skip = "Slow, only useful as a sanity check that the test codebase is sound and type-checks using the old method")>]
     let ``Test sequential type-checking`` (projectArgumentsFilePath: string) =
         testCompilerFromArgs Method.Sequential projectArgumentsFilePath
 
-    [<TestCaseSource(nameof localProjects)>]
-    [<Explicit("This only runs with the explicitly mentioned projects above")>]
+    [<MemberData(nameof localProjects)>]
+    [<Theory(Skip = "This should only run with the explicitly mentioned projects above")>]
```

Comment:
There is [no](https://stackoverflow.com/questions/37334245/can-you-mark-xunit-tests-as-explicit) `Explicit` attribute in Xunit (skipping the test by default, but running on-demand). It [should arrive](https://github.com/xunit/xunit/issues/2518) in Xunit 3.

Hence just marking as skipped. This anyway fails by default, without projects located on the disk.